plugins/woocommerce-blocks/bin/webpack-configs.js

```diff
@@ -986,6 +978,151 @@ const getInteractivityAPIConfig = ( options = {} ) => {
+			// @see https://github.com/webpack/webpack/issues/2329
+			// Using the ?ver string is needed here so the filename does not change between builds. The WordPress
+			// i18n system relies on the hash of the filename, so changing that frequently would result in broken
+			// translations which we must avoid.
+			// @see https://github.com/Automattic/jetpack/pull/20926
+			chunkFilename: `[name]-frontend${ fileSuffix }.js?ver=[contenthash]`,
+			filename: ( pathData ) => {
+				// blocksCheckout and blocksComponents were moved from core bundle,
+				// retain their filenames to avoid breaking translations.
+				if (
+					pathData.chunk.name === 'blocksCheckout' ||
+					pathData.chunk.name === 'blocksComponents'
+				) {
+					return `${ paramCase(
+						pathData.chunk.name
+					) }${ fileSuffix }.js`;
+				}
+
+				return `[name]-frontend${ fileSuffix }.js`;
+			},
+			uniqueName: 'webpackWcBlocksCartCheckoutFrontendJsonp',
+			library: [ 'wc', '[name]' ],
+		},
+		module: {
+			rules: [
+				{
+					test: /\.(j|t)sx?$/,
+					exclude: /node_modules/,
+					use: {
+						loader: 'babel-loader',
+						options: {
+							presets: [
+								[
+									'@wordpress/babel-preset-default',
+									{
+										modules: false,
+										targets: {
+											browsers: [
+												'extends @wordpress/browserslist-config',
+											],
+										},
+									},
+								],
+							],
+							plugins: [
+								isProduction
+									? require.resolve(
+											'babel-plugin-transform-react-remove-prop-types'
+									  )
+									: false,
+								'@babel/plugin-proposal-optional-chaining',
+								'@babel/plugin-proposal-class-properties',
+							].filter( Boolean ),
+							cacheDirectory: true,
+						},
+					},
+				},
+				{
+					test: /\.s[c|a]ss$/,
+					use: {
+						loader: 'ignore-loader',
+					},
+				},
+			],
+		},
+		optimization: {
+			concatenateModules:
+				isProduction && ! process.env.WP_BUNDLE_ANALYZER,
+			splitChunks: {
+				minSize: 200000,
+				automaticNameDelimiter: '--',
+				cacheGroups: {
+					commons: {
+						test: /[\\/]node_modules[\\/]/,
+						name: 'cart-checkout-vendors',
+						chunks: 'all',
+						enforce: true,
+					},
+					base: {
+						// A refined include blocks and settings that are shared between cart and checkout that produces the smallest possible bundle.
+						test: /assets[\\/]js[\\/](settings|previews|base|data|utils|blocks[\\/]cart-checkout-shared|icons)|packages[\\/](checkout|components)|atomic[\\/]utils/,
+						name: 'cart-checkout-base',
+						chunks: 'all',
+						enforce: true,
+					},
+					...getCacheGroups(),
+				},
+			},
+			minimizer: [
+				new TerserPlugin( {
+					parallel: true,
+					terserOptions: {
+						output: {
+							comments: /translators:/i,
+						},
+						compress: {
+							passes: 2,
+						},
+						mangle: {
+							reserved: [ '__', '_n', '_nx', '_x' ],
+						},
+					},
+					extractComments: false,
+				} ),
+			],
+		},
```

Comment:
I assume this was always there?

Reply:
> I assume this was always there?

@senadir which part? I did copy paste this from the front-end bundle because the configurations make sense to be exactly the same. The duplication is not ideal, and I think we could do some refactoring of the shared webpack plugins in future.

Reply:
> The duplication is not ideal, and I think we could do some refactoring of the shared webpack plugins in future.

+1 to this. While probably not a blocker of this PR, I would really like that everything that needs to be the same between the frontend bundle and the Cart & Checkout bundle is stored in the same place. Otherwise we risk introducing changes to one bundle and not the other.

Reply:
@Aljullu Yes you're right, I'm quite happy to add a follow up task to tackle refactoring the webpack configs. I'll add an issue for that.

Reply:
I've created an issue to deal with that: https://github.com/woocommerce/woocommerce/issues/48968

Reply:
> I did copy paste this from the front-end bundle because the configurations make sense to be exactly the same.

Yeah that was my question, if it's a new code or existing one moved there.